### DISCUSSION TITLE
Unable to clone Rebound 11

### DISCUSSION THREAD

**Discussion by errortek** [IviriusCommunity/Rebound#77] [General]
### 📄 Description

I've been trying to clone the Rebound 11 repository yesterday, but was unable to, due to the presence of LFS files. No files in the repo are over 100MB, so I feel that it would be safe to remove it.

### 🪜 Steps To Reproduce

1. Clone the repository
2. See LFS error

### 🤔 Expected behavior

Repo clones and you are able to compile it, edit files etc.

**Comment by Lamparter** [comment]
(as per gc on discord) 
Yeah this'll be fixed by the new build system which is coming soon™️

**Reply by errortek** [reply]
> (as per gc on discord) Yeah this'll be fixed by the new build system which is coming soon™️

Yes I know. I submitted it as a temporary fix which would allow people to clone it in the meantime.

**Reply by Lamparter** [reply]
Strange that some git clients don't let you clone without lfs files...
